README.md

```diff
@@ -48,16 +48,17 @@ Directly from unpkg.com
 
 # Banner Attributes
 
-| Name                   | Type            | Description                                                        |
-| ---------------------- | --------------- | ------------------------------------------------------------------ |
-| width                  | Number          | Banner width                                                       |
-| height                 | Number          | Banner height                                                      |
-| id                     | String          | The slot ID for this banner                                        |
-| category-id*           | Optional String | The category ID of the current page                                |
-| category-ids*          | Optional String | Comma (,) separated list of category IDs, the item must match all  |
-| category-disjunctions* | Optional String | Comma (,) separated list of category IDs, the item must match any  |
-| search-query           | Optional String | The search query of the current page                               |
-| location               | Optional String | The location for geotargeting                                      |
+| Name                   | Type            | Description                                                                 |
+| ---------------------- | --------------- | --------------------------------------------------------------------------- |
+| width                  | Number          | Banner width                                                                |
+| height                 | Number          | Banner height                                                               |
+| id                     | String          | The slot ID for this banner                                                 |
+| category-id*           | Optional String | The category ID of the current page                                         |
+| category-ids*          | Optional String | Comma (,) separated list of category IDs, the item must match all           |
+| category-disjunctions* | Optional String | Comma (,) separated list of category IDs, the item must match any           |
+| search-query           | Optional String | The search query of the current page                                        |
+| location               | Optional String | The location for geotargeting                                               |
+| target                 | Optional String | Define the target for the banner link (use `"_blank"` to open in a new tab) |
```

Comment:
I don't really like this interface because the problem we are trying to solve is boolean, you either open the link in current window or into a new tab. This gives the user options to put whatever they want here, and also means that the user must know that passing "_blank" opens a new tab.

Reply:
Also, what happens when the user implements their own banner rendering function?

Reply:
Boolean was my first thought, I'll revert to it 👍